src/swap/components/SwapProvider.tsx

```diff
@@ -95,6 +94,10 @@ export function SwapProvider({
 
   const { from, to } = useFromTo(address);
 
+  const { sendTransactionAsync } = useSendTransaction();
+
+  const config = useConfig();
```

Comment:
Like here
```
// Get wagmi config, to use with the waitForTransactionReceipt
const config = useConfig();
```

so as someone reads the code gets better understanding what's going on in a few months from now.

Reply:
added